src/components/RequestDashboard/SettingsSection.jsx

```diff
@@ -0,0 +1,271 @@
+      });
+    });
+  }, []);
+  const updateSetting = (key, value) => {
+    dispatch({
+      type: stateActions.updateSetting,
+      settingId: key,
+      value: value
+    });
+  };
+  const saveSettings = () => {
+    const headers = Object.keys(headerDefinitions).map(key => [key, state[key]]);
+    localStorage.setItem('reqgenSettings', JSON.stringify(headers));
+  };
+
+  const resetSettings = () => {
+    originalValues.forEach(e => {
+      try {
+        updateSetting(e[0], e[1]);
+      } catch {
+        console.log('Failed to reset setting value');
+      }
+    });
+  };
+  const clearQuestionnaireResponses =
+    ({ defaultUser }) =>
+    () => {
+      props.client
+        .request('QuestionnaireResponse?author=' + defaultUser, { flat: true })
+        .then(result => {
+          result.forEach(resource => {
+            props.client
+              .delete('QuestionnaireResponse/' + resource.id)
+              .then(result => {
+                console.log(result);
+              })
+              .catch(e => {
+                console.log('Failed to delete QuestionnaireResponse ' + resource.id);
+                console.log(e);
+              });
+          });
+        })
+        .catch(e => {
+          console.log('Failed to retrieve list of QuestionnaireResponses');
+          console.log(e);
+        });
+    };
+
+  const resetPims =
+    ({ pimsUrl }) =>
+    () => {
+      let url = new URL(pimsUrl);
+      const resetUrl = url.origin + '/doctorOrders/api/deleteAll';
+      console.log('reset pims: ' + resetUrl);
+
+      fetch(resetUrl, {
+        method: 'DELETE'
+      })
+        .then(response => {
+          console.log('Reset pims: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset pims error: ');
+          console.log(error);
+        });
+    };
+  const resetRemsAdmin =
+    ({ cdsUrl }) =>
+    () => {
+      let url = new URL(cdsUrl);
+      const resetUrl = url.origin + '/etasu/reset';
+
+      fetch(resetUrl, {
+        method: 'POST'
+      })
+        .then(response => {
+          console.log('Reset rems admin etasu: ');
+          console.log(response);
+        })
+        .catch(error => {
+          console.log('Reset rems admin error: ');
+          console.log(error);
+        });
+    };
+  const clearMedicationDispenses =
+    ({ ehrUrl, access_token }, consoleLog) =>
+    () => {
+      console.log('Clear MedicationDispenses from the EHR: ' + ehrUrl);
+      const client = FHIR.client({
+        serverUrl: ehrUrl,
+        ...(access_token ? { tokenResponse: access_token } : {})
+      });
+      client
+        .request('MedicationDispense', { flat: true })
+        .then(result => {
+          console.log(result);
+          result.forEach(resource => {
+            console.log(resource.id);
+            client
+              .delete('MedicationDispense/' + resource.id)
+              .then(result => {
+                consoleLog(
```

Comment:
When hitting the Clear EHR Medication Dispenses button it works, but then gets an error since consoleLog is not defined. Either replace this with console.log or make sure consoleLog is defined as expected. 
![Screenshot 2024-02-21 at 9 35 37 AM](https://github.com/mcode/request-generator/assets/31778437/c18a965b-5f03-4348-852d-7c008e34b65e)